wikizero.com,"HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)"
wima.ac.id,"HTTPSConnectionPool(host='wima.ac.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f10333b1d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='wima.ac.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f103305f700>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
winwin.rs,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
wixsite.com,"HTTPSConnectionPool(host='wixsite.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f10314f4910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='wixsite.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10348495e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
wordsense.eu,"HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)"
world.co.jp,"HTTPSConnectionPool(host='world.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f102fcf5070>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='world.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10349d6670>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
worldofbuzz.com,"HTTPSConnectionPool(host='worldofbuzz.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='worldofbuzz.com', port=443): Read timed out. (read timeout=2)"
www.gdansk.pl,"HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)"
www.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
xdomain.jp,"HTTPSConnectionPool(host='xdomain.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1030b52670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xdomain.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f103249be50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xn--12c1bik6bbd8ab6hd1b5jc6jta.com,"HTTPSConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f102cbf2e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1033a31ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xn--22c0bihcc9cwhcxj2ui.com,"HTTPSConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=80): Read timed out. (read timeout=2)"
xn--80acgfbsl1azdqr.xn--p1ai,"HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
xn--lamaana-7za.uy,"HTTPSConnectionPool(host='www.xn--lamaana-7za.uy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.xn--lamaana-7za.uy', port=443): Read timed out. (read timeout=2)"
xn--mgbkbq.xn--mgberp4a5d4ar,"HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))","HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))"
xn--sgb8bg.net,"HTTPSConnectionPool(host='xn--sgb8bg.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xn--sgb8bg.net', port=443): Read timed out. (read timeout=2)"
xn--titnjaa-o6a36e.hr,"HTTPSConnectionPool(host='xn--titnjaa-o6a36e.hr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='xn--titnjaa-o6a36e.hr', port=80): Read timed out. (read timeout=2)"
xn--y8jdm4g.com,"HTTPSConnectionPool(host='xn--y8jdm4g.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f102fe13df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xn--y8jdm4g.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1032483940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xoglasi.com,"HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)"
xtra.com.my,"HTTPSConnectionPool(host='www.xtra.com.my', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.xtra.com.my', port=443): Read timed out. (read timeout=2)"
y-lehti.fi,"HTTPSConnectionPool(host='www.y-lehti.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.y-lehti.fi', port=443): Read timed out. (read timeout=2)"
yamada-denkiweb.com,"HTTPSConnectionPool(host='yamada-denkiweb.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f103368c820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamada-denkiweb.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1033a31f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yamaguchi.lg.jp,"HTTPSConnectionPool(host='yamaguchi.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f103125a880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamaguchi.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f102b3d6a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yamano-music.co.jp,"HTTPSConnectionPool(host='yamano-music.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f10344405b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamano-music.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10332d75b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yatra.com,"HTTPSConnectionPool(host='www.yatra.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.yatra.com', port=443): Read timed out. (read timeout=2)"
yavitrina.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='yavitrina.ru', port=80): Read timed out. (read timeout=2)"
ybz.org.il,"HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)"
yeditepehastanesi.com.tr,"HTTPSConnectionPool(host='yeditepehastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f103209df40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yeditepehastanesi.com.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f102b267c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yellowpages.co.za,"HTTPSConnectionPool(host='yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
yellowpages.com,"HTTPSConnectionPool(host='yellowpages.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f102b1f4250>, 'Connection to yellowpages.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='yellowpages.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f102b230f40>, 'Connection to yellowpages.com timed out. (connect timeout=2)'))"
yellshops.com,"HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)"
yerevanmobile.am,"HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)"
yeutre.vn,"HTTPSConnectionPool(host='yeutre.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yeutre.vn', port=443): Read timed out. (read timeout=2)"
yhb.org.il,"HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)"
yodobashi.com,"HTTPSConnectionPool(host='yodobashi.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1032da8070>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yodobashi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10326b0370>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yokohama.lg.jp,"HTTPSConnectionPool(host='yokohama.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f102b252bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yokohama.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1034fb83a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ypfb.gob.bo,"HTTPSConnectionPool(host='ypfb.gob.bo', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1031465a00>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ypfb.gob.bo', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f10340f38b0>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))"
yubox.com,"HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
zabytek.pl,"HTTPSConnectionPool(host='zabytek.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='zabytek.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
zagonka.net,"HTTPSConnectionPool(host='zagonka.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1031fa2790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='zagonka.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10326e7040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
zalando.at,"HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)"
zalando.be,"HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)"
zalando.ch,"HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)"
zalando.co.uk,"HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)"
zalando.cz,"HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)"
zalando.de,"HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)"
zalando.dk,"HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)"
zalando.es,"HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)"
zalando.fi,"HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)"
zalando.fr,"HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)"
zalando.ie,"HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)"
zalando.it,"HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)"
zalando.nl,"HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)"
zalando.no,"HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)"
zalando.pl,"HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)"
zalando.se,"HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)"
zalora.co.id,"HTTPSConnectionPool(host='www.zalora.co.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.co.id', port=80): Read timed out. (read timeout=2)"
zalora.com.my,"HTTPSConnectionPool(host='www.zalora.com.my', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.com.my', port=80): Read timed out. (read timeout=2)"
zalora.sg,"HTTPSConnectionPool(host='www.zalora.sg', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.sg', port=80): Read timed out. (read timeout=2)"
zaobao.com.sg,"('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))","('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))"
zapweb.co.il,"HTTPSConnectionPool(host='www.rest.co.il', port=443): Max retries exceeded with url: /rest/80041163/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.rest.co.il', port=443): Max retries exceeded with url: /rest/80041163/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
zdravniki-zobozdravniki.net,"HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)"
zilinskazupa.sk,"HTTPSConnectionPool(host='zilinskazupa.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f102cae3b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='zilinskazupa.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f10319158e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ziveobce.cz,"HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))","HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))"
